templates/acp.pug

```diff
@@ -5,6 +5,7 @@ html(lang="en")
     +head()
     link(rel="stylesheet", type="text/css", href="/css/acp.css")
   body
+    include ../www/svg/coolhole-icons.svg
```

Comment:
oh one more, does acp need icons? I can't even get ACP to work...
![Uploading image.png…]()

Reply:
That screenshot didn't upload, but I can check that myself in a bit. If it isn't working we can just drop it for now. As I mentioned above, I'll see if placing the `svg` include in head works.

Reply:
oh annoying. here's the screenshot. and yea heard
![image](https://github.com/user-attachments/assets/1241f1fe-0fab-4d96-8968-d006a80300aa)